src/makeTxn.ts

```diff
@@ -192,7 +195,8 @@ export function makeKeyRegistrationTxnWithSuggestedParams(
   voteKeyDilution: any,
   suggestedParams: any,
   rekeyTo?: any,
-  nonParticipation = false
+  nonParticipation = false,
```

Comment:
nit: does this need to have a default value? it looks like that value is always set in the other constructors.

Reply:
This default value has runtime implications (i.e. JS will fill in `false` if no value is provided when the function is called) while the other definitions are purely for static type analysis